Sources/NostrSDK/Events/Tags/RelayTagParameter.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  RelayTagParameter.swift
+//
+//
+//  Created by Terry Yiu on 11/15/23.
+//
+
+import Foundation
+
+/// A relay tag parameter. This parameter may appear as metadata to various tags, such as pubkey tags and event tags.
+public protocol RelayTagParameter {
+    var relay: URL? { get }
```

Comment:
`relayURL`?

Reply:
This was a great suggestion. I was able to change `RecommendServerEvent` to conform to the protocol as well.